azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -169,10 +168,12 @@ private void finishLogin() {
      * @return Mono = true if this account is available
      */
     private Mono<Boolean> loginStep1() {
```

Comment:
method name should better describe what this method does, but not where this method is called.